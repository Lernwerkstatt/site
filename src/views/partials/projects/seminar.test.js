const workshop = require("./seminar.hbs");

describe("seminar.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      workshop({
        name: "Workshop",
        link: "http://www.die-lernwerkstatt.org/",
        text: "This is a workshop"
      })
    ).toMatchSnapshot();
  });
});
