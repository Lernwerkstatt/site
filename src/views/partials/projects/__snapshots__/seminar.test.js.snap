// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`seminar.hbs should match the snapshot 1`] = `
"<div class=\\"card\\">
  <div class=\\"card-body\\">
    <a rel=\\"noopener\\" href=\\"http://www.die-lernwerkstatt.org/\\" target=\\"_blank\\">
      <h5 class=\\"card-title\\">Workshop</h5>
    </a>
    <p class=\\"card-text\\">This is a workshop</p>
  </div>
</div>
"
`;
